model.go

```diff
@@ -1234,18 +1234,18 @@ func (m *model) validateStorage(validationCtx *validationContext) error {
 
 // validateSubnets makes sure that any spaces referenced by subnets exist.
 func (m *model) validateSubnets() error {
-	spaceIDs := set.NewStrings()
+	spaceUUIDs := set.NewStrings()
 	for _, space := range m.Spaces_.Spaces_ {
-		spaceIDs.Add(space.Id())
+		spaceUUIDs.Add(space.UUID())
 	}
 	for _, subnet := range m.Subnets_.Subnets_ {
-		// space "0" is the new, in juju 2.7, default space,
-		// created with each new model.
-		if subnet.SpaceID() == "" || subnet.SpaceID() == "0" {
+		// TODO(nvinuesa): How will we identify the alpha space
+		// in UUID?
+		if subnet.SpaceUUID() == "" {
 			continue
 		}
```

Comment:
IMO we shouldn't be doing these checks at this level, this should be a serialization/deserialization library and with these validate functions we are adding business logic. This particular check is not OK anymore and we should decide if we want to add the (new) alpha space UUID here or remove this validate altogether. @wallyworld @SimonRichardson

Reply:
after discussing with @manadart, we are keeping `0` as the uuid for the alpha space.